src/components/Footer/Footer.jsx

```diff
@@ -1,12 +1,14 @@
 import React from 'react';
 import styles from './Footer.module.css'; 
 import HeroFooter from './HeroFooter';
+import SubFooter from './SubFooter';
 
 
 const Footer = () => {
   return (
     <footer className={styles.footer}>
        <HeroFooter/>
```

Comment:
HeroFooter should not be here any more as it is missing in the Contacts page

Reply:
HeroFooter _could_ be here, but it would need to render conditionally, only rendering if the current page is NOT the contact page. This would involve using react-router and/or react-router-dom (I'm not sure offhand about implementation), and is the way I would do it if there was any possibility of the site growing.

The other option is it could be rendered directly by the Home and About page components. This means maintaining the component props in two separate places, but it is a very quick way to get things up and running, and on a project this size, it's managable.

Reply:
@joanwolk I have made the necessary changes in the footer section